src/main/java/de/rytrox/gamestate/GameState.java

```diff
@@ -23,6 +24,7 @@ public GameState(String identifier, Listener... listeners) {
 
     public void registerListener(Listener listener) {
         this.registeredListener.add(listener);
+        HandlerList.unregisterAll(listener);
```

Comment:
Ähm, falsche Methode. Das muss in die unregisterListener Methode 😂

Reply:
fick dich xD